src/infra/archive-stable-version-installers.md

```diff
@@ -0,0 +1,20 @@
+# Archive of Rust Stable Standalone Installers
+
+The official Rust standalone installers contain a single release of Rust, and
+are suitable for offline installation. They come in three forms: tarballs
+(extension `.tar.gz`), that work in any Unix-like environment, Windows
+installers (`.msi`), and Mac installers (`.pkg`). These installers come with
+`rustc`, `cargo`, `rustdoc`, the standard library, and the standard
+documentation, but do not provide access to additional cross-targets like
+`rustup` does.
+
+The most common reasons to use these are:
+
+- Offline installation
+- Preferring a more platform-integrated, graphical installer on Windows
+
+Each of these binaries is signed with the [Rust signing key], which is
+[available on keybase.io], by the Rust build infrastructure, with [GPG]. In the
+tables below, the `.asc` files are the signatures.
```

Comment:
Just FYI, these links are all not working.

It would probably be ideal to instead of copying all this text, to use a `{{#include …}}` to have both pages share this from a common file.

Reply:
Done.

Reply:
It looks like there is still a lot of duplicated content between the two pages. What I was thinking is putting everything from "The official Rust standalone..." down to "In the tables below, the `.asc` files are the signatures." into a shared file (likes 3 to 18).

Reply:
Done.